src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -671,12 +691,26 @@ def iot_hub_get_stats(client, hub_name, resource_group_name=None):
     resource_group_name = _ensure_resource_group_name(client, resource_group_name, hub_name)
     return client.iot_hub_resource.get_stats(resource_group_name, hub_name)
 
+def validate_authentication_type_input(endpoint_type, connection_string=None, authentication_type=None, endpoint_uri=None, entity_path=None):
+    print('validating...')
```

Comment:
Is this a temporary `print` for debugging? You could put various messages in debug logs if it helps anyone see what's going on.

Reply:
Yup, that's simply a debug line - will remove.